cypress/integration/ui/menu.spec.js

```diff
@@ -14,24 +14,26 @@ describe('Menu', () => {
         expect(menu[2].items[1].items[3].title).to.equal('Virtual Machines');
       });
 
-    cy.menu('Overview')
+    cy.menu('Overview', 'Dashboard')
       .get('widget-wrapper');
 
-    cy.menu('Services')
+    cy.menu('Overview', 'Reports')
+      .expect_explorer_title('All Saved Reports');
+
+    cy.menu('Services', 'My Services')
       .expect_explorer_title('Active Services');
 
-    cy.menu('Compute')
+    cy.menu('Compute', 'Clouds', 'Providers')
       .expect_show_list_title('Cloud Providers');
 
-    cy.menu('Overview', 'Reports')
-      .expect_explorer_title('All Saved Reports');
-
     cy.menu('Compute', 'Infrastructure', 'Virtual Machines')
       .expect_explorer_title('All VMs & Templates');
 
+    // FIXME:  This isn't valid functionality for the menu at this time.
```

Comment:
you can probably remove this, as you fixed it on line 20 :)

Reply:
@himdel so I left this here since I wasn't sure if it was the intent to remove this feature in https://github.com/ManageIQ/manageiq-ui-classic/pull/6963 or if it was an oversight.  If it was the intent to remove the following:

```
     // test it remembers the last Overview > * screen used
```

Then I am file getting rid of this section.

Reply:
Oh it was intentional, the whole "remember the last active menu item per section" feature is gone, clicking on sections now opens the submenu.

Reply:
Okay, sorry for the delay on this.  Should be fixed now.